rollup.config.js

```diff
@@ -52,6 +53,9 @@ module.exports = ({
     // 'ts-pattern',
   ],
   plugins: [
+    json({
+      compact: true,
```

Comment:
Not to make the package.json file impact on the bundle size 
https://linguinecode.com/post/rollup-plugin-json-import